.github/workflows/lint.yml

```
@@ -8,13 +8,20 @@ jobs:
         runs-on: ubuntu-latest
         steps:
             - uses: actions/checkout@v4
-            # python is required for the "pre-commit" action.
-            - uses: actions/setup-python@v5
             - name: Setup Node
               uses: actions/setup-node@v4
               with:
                   node-version: "18"
                   cache: "npm"
+
             - name: npm install
               run: npm ci --silent
-            - uses: pre-commit/action@v3.0.1
+
+            - name: Check Lint
+              run: npm run lint:check
+
+            - name: Check Formatting
+              run: npm run format:check
+
+            - name: Check Lua Styling
+              run: npm run stylua:check
```

formatting is just another kind of lint. "check" is redundant. 

actually, why do these need names? the script name is more useful than these other names.

```suggestion
            - run: npm run lint:check
            - run: npm run format:check
            - run: npm run stylua:check
```

---

The names are just easier to grok sometimes, but I don't have a strong preference. If it helps people realize what commands they actually are, it's A-OK to me